### PR TITLE
Environment: processing no longer necessary

### PR DESCRIPTION
**Pull request by bryanlatten**
Just flagged php-fpm to not wipe the environment

### CODE REVIEW

File: container/root/etc/php5/fpm/conf.d/web.user.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ; Use this file to override specific defaults in the stock Ubuntu php.ini
-; NOTE: this only applies to web requests through Apache, not CLI (worker,cron) instantiations
+; NOTE: this only applies to web requests through the web server, not direct PHP instantiations like a background or cron task
```
Comment (bryanlatten) [inline container/root/etc/php5/fpm/conf.d/web.user.ini:2]: just removing mention of apache :)

**Comment by jimdelois** [comment]
Wow.

**Comment by jimdelois** [comment]
:white_check_mark:

File: container/root/run.d/99-phpfpm.sh [hunk 2]
```diff
@@ -17,6 +17,9 @@ sed -i "s/pm.max_spare_servers = [0-9]\+/pm.max_spare_servers = 128/" $PHPFPM_CO
 
 sed -i "s/;pm.max_requests = [0-9]\+/pm.max_requests = 1024/" $PHPFPM_CONF
 
+# Ensure environment variables aren't cleaned, will make it into FPM  workers
+sed -i "s/;clear_env/clear_env/" $PHPFPM_CONF
```
Comment (chrishenry) [inline container/root/run.d/99-phpfpm.sh:21]: Nice and clean

**Comment by chrishenry** [comment]
:+1:

File: container/root/run.d/99-phpfpm.sh (hunk 2 above)
Comment (bossjones) [inline container/root/run.d/99-phpfpm.sh:21]: just double-checking, will this match more than one occurrence of this pattern? do we need a `/g` to make sure it matches more?

**Comment by mannytoledo** [comment]
:+1:

File: container/root/run.d/99-phpfpm.sh (hunk 2 above)
Comment (bryanlatten) [inline container/root/run.d/99-phpfpm.sh:21]: @bossjones I would only want it to match once? no?

**Pull request merged**